decentraland/explorer-desktop
# fix: removed color grading from graphics widget

## What does this PR change?

Settings ui was not being loaded because the last version of unity-renderer has color grading deleted. This changes fixes the issue.

## How to test the changes?

1. Press TAB and open settings menu
2. Go to graphics
3. All settings should be displayed correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
